js/piwik.js

```diff
@@ -1138,15 +1138,7 @@ if (typeof Piwik !== 'object') {
             }
 
             function _elementInDocument(element) {
-                element = element.parentNode;
-
-                while (element) {
-                    if (element === documentAlias) {
-                        return true;
-                    }
-                    element = element.parentNode;
-                }
-                return false;
+                return documentAlias.documentElement.contains(element);
```

Comment:
[It works since IE5 :)](https://github.com/piwik/piwik/pull/8637#issuecomment-134879548)